Kitodo-API/pom.xml

```diff
@@ -28,4 +28,12 @@
     <artifactId>kitodo-api</artifactId>
     <name>Kitodo - API</name>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
You are sure about this artifact id? In the moved file I see no class usage for `slf4j` or did I missed something?